scripts/comptime_align.py

```diff
@@ -1,14 +1,19 @@
 #!/usr/bin/python3
 import sys
 import os
+import shutil
 
 v1_file = sys.argv[1]
 v2_file = sys.argv[2]
 threshold = 0.001
 
+if not os.path.exists(v1_file) and not os.path.exists(v2_file):
+    exit(1)
```

Comment:
```suggestion
if not os.path.exists(v2_file):
    exit(1)
```
Merge this logic with the code in line 15-16.